students/morgan/dnd_project/bin/generator.py

```diff
@@ -0,0 +1,65 @@
+from encounter_builder.interface import ui
+from encounter_builder.data_models import party_data, monster_data
+from encounter_builder.util import number_crunchers
+# import setup
+# import encounter_builder.resource_files
+
+
+
+if __name__ == "__main__":
+
+
+
+    # xp_budget_file = setup.package_data['encounter_budget_by_level']
+    xp_budget_file = "encounter_builder/resource_files/xp_budget_by_level.csv"
+    monster_multiplier = "encounter_builder/resource_files/monster_multiplier.csv"
+    xp_by_cr = "encounter_builder/resource_files/xp_by_cr.csv"
+    cr_guide = "encounter_builder/resource_files/cr_guide.csv"
+    # print(xp_budget_file)
+
+    the_party = party_data.Party(ui.Get_PCs())
+    # print(the_party.members, the_party.min, the_party.crowd)
+
+    _ = number_crunchers.establish_budgets(xp_budget_file, the_party)
```

Comment:
usually "_" is used as a name meaning "I don't need this at all", if you need a short term temp variable, still use a meaningful name.